data/json/mapgen/homeimprovement_superstore.json

```diff
@@ -60,6 +60,11 @@
       [ "paint_brush", 55 ]
     ]
   },
+  {
+    "id": "carpet",
+    "type": "item_group",
+    "items": [ [ "r_carpet", 100 ], [ "p_carpet", 100 ], [ "y_carpet", 100 ], [ "g_carpet", 100 ], [ "p_carpet", 100 ] ]
+  },
```

Comment:
Could you move all these item groups to the itemgroups folder, maybe to supplies.json? We should get away from defining item groups in map files, they get lost and nobody updates them

Reply:
I'll try my best. I don't know if all of these item groups belong to ```supplies_json```.